src/BentoMenu.tsx

```diff
@@ -0,0 +1,11 @@
+import { BentoMenuBox } from "./BentoMenuBox";
+
+export const BentoMenu = () => {
+	return (
+		<div>
+			<BentoMenuBox text="Filming" to="/filming" />
+			<BentoMenuBox text="Photography" to="/photography" />
+			<BentoMenuBox text="Equipment Rental" to="/equipment-rental" />
```

Comment:
Už ze začátku je dobrý myslet na reusability, takže třeba tyhle linky by mohly být někde globálně jako constanty a pak tady + v routeru na ně jen odkazovat, aby je stačilo měnit jen na jednom místě a promítlo se to v celé appce. Jinak se ti hlavně třeba u těch linků pak bude stávat, že ti nějaké tlačítko nebude fungovat a nebudeš o tom 2 měsíce vědět.

Něco jako:
```
export const ROUTES = {
  film: "/filming",
  photo: "/photography",
  rental: "/equipment-rental"
} as const
```
A pak `<BentoMenuBox text="Filming" to={ROUTES.film} />`.